Create 2021.35 release notes (#350)
Crunch-io/ta-da: content/release-notes/2021-09-20.md

+++
date = "2021-09-20T10:00:00-07:00"
publishdate = "2021-09-20T10:00:00-07:00"
draft = false
title = "Release 2021.35"
images = ["https://crunch.io/img/logo-1200x630.png"]
+++

### Crunch web application

- Fixed an issue that occurred when a tab on a dashboard tile was not allowing the user to change the selection.
- Improved the behavior in the 'Dashboards Group' panel in situations when one or more of the filter variables have been deleted from the dataset.
- Fixed an issue that occurred when selecting a category in the 'Dashboard Group' panel that sometimes did not reflect in the rendering of the dashboard.
- Fixed a bug in [Confirmit integrations](https://help.crunch.io/hc/en-us/articles/4403966631309-How-your-survey-metadata-maps-from-Confirmit-into-Crunch) when response data was smaller than expected.
- Corrected an issue that occurred when a dashboard tile no longer reflects the column selected for hypothesis testing comparisons after changing the selection in the tab pulldown within that tile.

> - Added a new option in the 'Dashboard Edit Groups' panel for removing a filter that corresponds to a variable that has been deleted from the dataset.

Reviewer: ```suggestion
- Added the ability to remove a stale reference in dashboard ‘groups’ when a variable has been deleted from the dataset.
```

> - Added the ability to download a Crunch Automaton script from the UI when creating a folder structure in a dataset and moving variables into it (a link appears allowing you to download the script).

Reviewer: ```suggestion
- Added the ability to download a Crunch Automaton script (a series of `organize` commands) reflecting the current variable folder structure from the ‘Edit variables’ interface.
```

Author: Updated.
